plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -766,6 +779,57 @@ export class EventsProcessor {
         }
     }
 
+    private tryStoreSessionRecordingToObjectStorage(
```

Comment:
So this class is doing way too much. Please put this elsewhere, e.g. extract class/method/etc - we don't want this class to be also responsible for business logic of object storage.